File: warden_web/app/models/test_case.rb
class TestCase < ActiveRecord::Base
  attr_accessible :feature_file_path, :feature_name, :last_run_at,
    :name, :register_at, :tc_id, :warden_project_id, :warden_project
  belongs_to :warden_project
  has_many :test_case_run_infos


end

File: warden_web/public/app_js/model/test_case.js
Ext.define('WardenWeb.model.test_case', {
  extend: 'Ext.data.Model',
  fields: ['feature_name', 'name', 'feature_file_path', 'tc_id', 'register_at',
    { name: 'project_name', type: 'string', mapping: 'warden_project.name' }
  ]
});
